specification/automation/resource-manager/readme.md

```diff
@@ -196,6 +231,15 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 output-folder: $(go-sdk-folder)/services/automation/mgmt/2017-05-15-preview/automation
 ```
 
+### Tag: package-2018-01-preview and go
+
+These settings apply only when `--tag=package-2018-01-preview --go` is specified on the command line.
+Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+``` yaml $(tag) == 'package-2018-01-preview' && $(go)
+output-folder: $(go-sdk-folder)/services/automation/mgmt/2018-01-15-preview

```

Comment:
>/ [](start = 40, length = 1)

Add a preview folder here :D

Reply:
I don't see other services adding a `preview` folder or a preview folder showing up in the Go SDK repo.

Example - [Data Migration](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/compute/resource-manager#tag-package-container-service-2015-11-preview-and-go) and [Compute](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/compute/resource-manager#tag-package-container-service-2015-11-preview-and-go). Do I need to add this then?

Reply:
There is not problem if the golang section is left out. We will figure it out later

Reply:
What do you suggest then, for getting the PR through? I can send a separate PR for the same.